cmd/network_test.go

```diff
@@ -258,23 +259,29 @@ nw1 network-1
 				args := []string{"network", "update", *want.ID,
 					"--description", want.Description,
 					"--add-prefixes", "prefix",
-					"--remove-prefixes", "z",
+					"--remove-prefixes", "a,b",
 					"--add-destinationprefixes", "dest",
-					"--remove-destinationprefixes", "z",
+					"--remove-destinationprefixes", "y,z",
 				}
 				assertExhaustiveArgs(t, args, "file")
 				return args
 			},
 			mocks: &client.MetalMockFns{
 				Network: func(mock *mock.Mock) {
-					networkToUpdate := network1
-					networkToUpdate.Prefixes = []string{"z"}
-					networkToUpdate.Destinationprefixes = []string{"z"}
-					// networkToUpdate.Description = "Old description"
+					networkToUpdate := mustJsonDeepCopy(t, network1)
+					networkToUpdate.Prefixes = []string{"a", "b"}
+					networkToUpdate.Destinationprefixes = []string{"y", "z"}
+					networkToUpdate.Description = "Old description"
+
 					mock.On("FindNetwork", testcommon.MatchIgnoreContext(t, network.NewFindNetworkParams().WithID(*network1.ID)), nil).Return(&network.FindNetworkOK{
 						Payload: networkToUpdate,
 					}, nil)
-					mock.On("UpdateNetwork", testcommon.MatchIgnoreContext(t, network.NewUpdateNetworkParams().WithBody(networkResponseToUpdate(network1))), nil).Return(&network.UpdateNetworkOK{
```

Comment:
It would only be possible to use `networkResponseToUpdate` if the user could modify all possible fields in the update request.